pontoon/base/templates/widgets/heading_info.html

```diff
@@ -130,3 +130,21 @@
   </div>
 </div>
 {% endmacro %}
+
+{% macro item_download_button(locale,slug,class='') %}
+<div class="download-selector select">
+  <div class="button selector">
+    <i class="fa fa-cloud-download-alt"></i>
+  </div>
+  <div class="{{ class }}" >
```

Comment:
Why do you pass class as a parameter, in both cases with the same value ("menu")? Seems like you could just hardcode this to "menu".